README.md

```diff
@@ -36,7 +36,7 @@ If anything feels off, or if you feel that some functionality is missing, please
 
 ### Linux
 
-`gh` is available via [Homebrew](#homebrew), and as downloadable binaries from the [releases page][].
```

Comment:
Was this a bad link? As far as I understand, this linked to the "Homebrew" section where the `brew install gh` instructions are.

Reply:
Yes, that is true.

The link is to the anchor on the Homebrew Install/Upgrade commands in the macOS Installation section. The actual link to https://brew.sh/ may be offscreen (as it was for me) until scrolling up a bit to reveal the link to the site above the anchor the user is directed to when clicking the Homebrew link in the Linux Installation section.

I was a bit confused when I input the command in my Linux terminal and was directed to install the unofficial GH CLI. But I figured it out after I returned to the ReadMe and scrolled up to see the link to the Homebrew website.

I proceeded to install Homebrew, but while I was waiting for the nearly 500 mb download to complete, I followed the link to the "more information and distro-specific instructions" in the Linux install docs where I found there was a repo for Ubuntu I could use instead of Homebrew.

It would be better if the Linux instructions stood apart on their own without referring to the macOS instructions. It may good to more prominently indicate the availability of `gh` in distribution package management repositories rather than listing Homebrew as the first option for Linux. But this PR wouldn't change any of that, so I'll close it.